### PR TITLE
Add main menu

### PR DESCRIPTION
**Pull request by sandorbence**

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -7,8 +7,14 @@
...
+    <button class="menu-btn" id="settings">Settings</button>
+    <button class="menu-btn" id="high-scores">High scores</button>
+    <button class="menu-btn" id="about">About</button>
+  </div>
   <canvas></canvas>
   <script src="main.js"></script>
 </body>
 
-</html>
+</html>
```
Comment (s4rd3d) [inline dist/index.html:20]: Please add an empty line at the end of the file.
You can usually configure your editor to add the empty line on save, so you don't have to worry about it.

File: src/index.js [hunk 2]
```diff
@@ -12,8 +12,8 @@ canvas.height = CANVAS_HEIGHT;
 
 // Generate a new world
 const world = new World();
-world.generate();
 
 // Create a new renderer and start the animation
 const render = new Render(canvas, world);
 render.animate();
+world.generate();
```
Comment (s4rd3d) [inline src/index.js:19]: Why did you move this line here?
It makes more sense to me to first add objects to the world, then start animating it.

File: src/style/style.css [hunk 3]
```diff
@@ -1,7 +1,46 @@
...
+
+.hidden {
+  visibility: hidden;
+}
+
+.focused {
+  opacity: 1;
+  border-width: 5px;
+  border-color: white;
+}
```
Comment (s4rd3d) [inline src/style/style.css:46]: Please add a new line at the end of the file.

File: src/style/style.css [hunk 4]
```diff
@@ -1,7 +1,46 @@
...
+  margin: 5px;
+  background-color: transparent;
+  opacity: 0.5;
+  width: 95%;
+  padding: 20px;
+  border-radius: 10px;
+  font-size: 3rem;
+  color: white;
+}
+
```
Comment (s4rd3d) [inline src/style/style.css:37]: Maybe we could add focus to buttons when they are hovered. It creates a smoother feeling when navigating the menu with the mouse:

```css
.menu-btn:hover {
  opacity: 1;
  border-width: 5px;
  border-color: white;
}
```

File: src/index.js (hunk 2 above)
Comment (sandorbence) [reply src/index.js:19]: No idea how it got swapped, corrected.

File: src/style/style.css (hunk 4 above)
Comment (sandorbence) [reply src/style/style.css:37]: I added it as a "mouseover" event listener instead. This way we did not need to copy already existing css rules, also it does not mess with the script handling the focus (adding as css does).

**Review by s4rd3d — APPROVED** [review]
